src/main/java/ch/njol/skript/classes/data/DefaultOperations.java

```diff
@@ -115,6 +115,9 @@ public class DefaultOperations {
 		Arithmetics.registerOperation(Operator.SUBTRACTION, Date.class, Timespan.class, Date::minus);
 		Arithmetics.registerDifference(Date.class, Timespan.class, Date::difference);
 
+		// String + String
```

Comment:
```suggestion
		// String - String
```
The comment describes the type of the two operands, not the operation